### PR TITLE
[IMP] mail: turn 'Message/prettyBodyAsMarkup' to field

### PR DESCRIPTION
**Pull request by alexkuhn**
Task-2793280

### CODE REVIEW

File: addons/mail/static/src/models/message_in_reply_to_view/message_in_reply_to_view.js [hunk 1]
```diff
@@ -29,6 +31,13 @@ registerModel({
             }
             parentMessageView.update({ doHighlight: true });
         },
+        /**
+         * @private
+         * @returns {Markup}
+         */
+        _computeContentAsMarkup() {
+            return markup(this.messageInReplyToView.messageView.message.parentMessage.prettyBody);
+        },
```
Comment (seb-odoo) [inline addons/mail/static/src/models/message_in_reply_to_view/message_in_reply_to_view.js:40]: `markup` should be produced as close as possible to the original source of the "injection"
In this case, it should be directly into `prettyBody`, or worse case as a `prettyBodyMarkup` field.

**Review by oomsveta — APPROVED** [review]

**Comment by alexkuhn** [comment]
@seb-odoo need ci/security approval

**Comment by seb-odoo** [comment]
@robodoo override=ci/security
PR is just a move. Unsafe usage of `markup`, but `prettyBody` is considered safe (coming from sanitized Html field in python, and post-transformed "safely" in JS).

**Comment by alexkuhn** [comment]
@robodoo r+

**Pull request closed without merging**